boards/b-l072z-lrwan1/include/periph_conf.h

```diff
@@ -25,7 +25,7 @@
 #endif
 
 #include "periph_cpu.h"
-#include "l0l1/cfg_clock_default.h"
+#include "clk_conf.h"
```

Comment:
How about

```suggestion
#include "clk_conf_default.h"
```

so it's clear this is not the definite config but custom boards could use a custom config instead (e.g. not use the maximal CPU frequency to save energy)

Reply:
> custom boards could use a custom config instead

This is still possible by defining the constants before the includes (as done for some other STM32 boards). The `clk_conf.h` is not something that should re-done in custom boards, it contains the clock configuration definitions (with default values indeed but not only).

For a custom boards with different clock settings, developer are encouraged to either:
- define custom constants in the `periph_conf.h` header of the custom board,
- use CFLAGS
- use Kconfig => this will be the preferred solution once we switch to full Kconfig configuration (using .config files at board level)

So I wouldn't use `_default` suffix for clock configuration file names included in boards. Maybe the family headers filenames could be adapted.